README.md

```diff
@@ -29,12 +29,12 @@ This README is meant as a brief walkthrough on the core features of CDC Connecto
 
 We need several steps to setup a Flink cluster with the provided connector.
 
-1. Setup a Flink cluster with version 1.12+ and Java 8+ installed.
```

Comment:
What about creating an individual issue dropping Flink 1.13 support? Currently the commit is linked to #2403, whose description is about metrics. I think dropping support of a Flink version is relatively a big change and it deserves an individual issue to get tracked.

Reply:
This sounds good, and I will remove it and put it in an individual issue.